crates/matrix-sdk-crypto/src/olm/account.rs

```diff
@@ -1277,7 +1277,7 @@ impl Account {
                         // session in lieu of an OTK.)
 
                         warn!(
-                            session_id = session.session_id(),
+                            ?session,
```

Comment:
Do you want to keep this logging change?

Reply:
Thank you! Corrected and re-pushed along with a compile fix.